documentation/endpoints/posts.md

```diff
@@ -40,29 +40,21 @@ Example Response:
 ```
 {
   "data": {
-    "signup_id": 31,
```

Comment:
Totally! I just removed that since we're using all these Transformers now to populate the `/activity` endpoint instead of a separate `ActivityTransformer` and followed the structure that we wanted to be returned according to the wiki :)